external_libraries/oscpack_build.cpp

```diff
@@ -1,8 +1,8 @@
-#include <boost/detail/endian.hpp>
+#include <boost/predef/other/endian.h>
 
-#ifdef BOOST_LITTLE_ENDIAN
+#ifdef BOOST_ENDIAN_LITTLE_BYTE
 #define OSC_HOST_LITTLE_ENDIAN
-#elif defined(BOOST_BIG_ENDIAN)
+#elif defined(BOOST_ENDIAN_BIG_BYTE)
```

Comment:
can you please remove changes to this file? this is already covered by https://github.com/supercollider/supercollider/pull/4983, and what you have here is actually not correct (see that PR for more info)

Reply:
Done!